packages/server/prisma/seed.ts

```diff
@@ -0,0 +1,8 @@
+import { createMultipleUsers } from "../src/seeder/utils";
+
+const main = async () => {
+  const newUsers = await createMultipleUsers(14)
+  return newUsers
+};
+
+main().then(console.log).catch(console.log);
```

Comment:
We need to rethink the approach of invoking the main function directly. The original index.ts file, structured as a class, had the advantage of not running on server startup. Running migrations upon server startup is not our preferred method. Instead, we should trigger migrations once a user is created by our contributors. This approach allows us to focus on a specific user (their email ID) and ensures that all activities revolve around this user as the central point. This user-centric strategy aligns better with our goals.